### PR TITLE
Added note for private endpoint limitations

### PR DESCRIPTION
**Pull request by lanicolas**
Added a note for private endpoint limitations on connection policy as it is mentioned here https://docs.microsoft.com/en-us/azure/azure-sql/database/private-endpoint-overview?view=azuresql#limitations, otherwise it is assumed that you can use any of them independently if you use private link or not.

### CODE REVIEW

File: azure-sql/database/connectivity-architecture.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ Servers in SQL Database and Azure Synapse support the following three options fo
 
 We highly recommend the `Redirect` connection policy over the `Proxy` connection policy for the lowest latency and highest throughput. However, you will need to meet the additional requirements for allowing network traffic as outlined above. If the client is an Azure Virtual Machine, you can accomplish this using Network Security Groups (NSG) with [service tags](/azure/virtual-network/network-security-groups-overview#service-tags). If the client is connecting from a workstation on-premises then you may need to work with your network admin to allow network traffic through your corporate firewall.
 
+> [!IMPORTANT]
+> Connections to private endpoint only support **Proxy** as the [connection policy](connectivity-architecture.md#connection-policy)
```
Comment (Jak-MS) [inline azure-sql/database/connectivity-architecture.md:52]:
```suggestion
> Connections to private endpoint only support **Proxy** as the [connection policy](connectivity-architecture.md#connection-policy).
```

**Comment by Jak-MS** [comment]
@rohitnayakmsft 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rohitnayakmsft** [comment]
#sign-off

**Comment by PRMerger8** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**